src/main/java/org/jenkinsci/plugins/authorizeproject/ProjectQueueItemAuthenticator.java

```diff
@@ -93,23 +111,31 @@ public Authentication authenticate(Queue.Item item) {
         }
         return prop.authenticate(item);
     }
-    
-    public Map<String, Boolean> getStrategyEnabledMap() {

```

Comment:
Seems breaking but depends on usage.

Reply:
A getter for a now deprecated field could be added if really required to be 100% backward compatible.

Reply:
> if really required to be 100% backward compatible.

Binary compatibility is required.
Please add the getter.

Reply:
I just added it.